### DISCUSSION TITLE
IncrementDecrementSelector jump to specific non-consecutive bank?

### DISCUSSION THREAD

**Discussion by Nivolp1976** [tttapa/Control-Surface#575] [Q&A]
Hi there. I have a new silly question. It is possible to call a specific bank in IncrementDecrementSelector. For example, being on bank 1, jump to 10?

**Comment by tttapa** [comment]
I'm not sure what you mean, but you can use [`bank.select(9)`](https://tttapa.github.io/Control-Surface-doc/Doxygen/db/dbd/classBank.html#a801ab5dedea8200f1cdb26533cb6f78b) to select the 10th bank.  
When using a selector, you can use [`selector.set(9)`](https://tttapa.github.io/Control-Surface-doc/Doxygen/d1/d54/classSelector.html#a18f36819c7d33889ce08a322ace8c950).

**Reply by Nivolp1976** [reply]
Muchas gracias funciona a la perfeccion!!

**Reply by WeaponX15** [reply]
> I'm not sure what you mean, but you can use [`bank.select(9)`](https://tttapa.github.io/Control-Surface-doc/Doxygen/db/dbd/classBank.html#a801ab5dedea8200f1cdb26533cb6f78b) to select the 10th bank. When using a selector, you can use [`selector.set(9)`](https://tttapa.github.io/Control-Surface-doc/Doxygen/d1/d54/classSelector.html#a18f36819c7d33889ce08a322ace8c950).

How should the code be written to select a specific bank? Sorry but I tried to write it myself and it didn't work out, could you give me an example?

Thanks Peter.

**Reply by Nivolp1976** [reply]
Hello, does this work for you?
     `
 if (bank.getSelection() ==  3) {
      bank1.select(9) ;

      } else { bank1.select(0) ;
      }
`
